Samples/2.0/ServiceFabricMesh/VotingApp/VotingData/Program.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Orleans.Configuration;
+using Orleans.Hosting;
+using VotingContract;
+
+namespace VotingData
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var silo = new SiloHostBuilder()
+                .ConfigureEndpoints(siloPort: 30001, gatewayPort: 30002, listenOnAnyHostAddress: true)
+                .Configure<ClusterOptions>(options =>
+                {
+                    options.ClusterId = "dev";
+                    options.ServiceId = "votingapp";
+                })
+                .UseAzureStorageClustering(options => options.ConnectionString = Environment.GetEnvironmentVariable("CLUSTERING_CONNECTION_STRING"))
```

Comment:
Regarding https://github.com/dotnet/orleans/issues/5069, do you know what it would take to support clustering out-of-the-box inside Mesh without the need for external storage? Once standalone reliable collections land, that’s one option, but maybe not ideal? It somehow feels like it could be tighter integrated with Setvice Fabric.

Reply:
Standalone reliable collections will make it possible. I don't believe that Mesh has any suitable facilities for clustering yet - clustering currently has to guarantee some things around consistency which DNS isn't suited for.

I'll reply on the issue you opened (thanks for that)